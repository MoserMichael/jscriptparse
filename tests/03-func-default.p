

def add(num, inc=1)
    num + inc


println("increment: ", add(42))
println("addd: ", add(200,200))

