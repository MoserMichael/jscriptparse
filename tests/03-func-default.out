increment:  43
addd:  400
